Functions/xASL_bids_Dicom2JSON.m

```diff
@@ -416,20 +416,31 @@
 				% -----------------------------------------------------------------------------
 				
 				% Limit AcquisitionTime to one value
-				if  isfield(t_parms{parmsIndex},'AcquisitionTime')
-					for iL=1:length(t_parms{parmsIndex})
-						t_parms{parmsIndex}(iL).AcquisitionTime = xASL_str2num(t_parms{parmsIndex}(iL).AcquisitionTime);
-					end
-					
-					if length(t_parms{parmsIndex})>1
-						if imPar.bVerbose; fprintf('%s\n','Parameter AcquisitionTime has multiple values:'); end
-						%                     t_parms{parmsIndex}.AcquisitionTime
-						if imPar.bVerbose; fprintf('%s\n','using minumum value:'); end
-						%                     t_parms{parmsIndex}.AcquisitionTime = min(t_parms{parmsIndex}.AcquisitionTime)
-						tempAcquisitionTime = t_parms{parmsIndex}(1).AcquisitionTime;
-						t_parms{parmsIndex} = rmfield(t_parms{parmsIndex},'AcquisitionTime');
-						t_parms{parmsIndex}(1).AcquisitionTime = tempAcquisitionTime;
-					end
+                if parmsIndex>numel(t_parms)
+                    % This statement catches the cases where we iterate out of valid ranges of the t_parms struct. Right now I
+                    % mostly saw that bug during the import using bMatchDirectory = false! We probabaly determine the index wrong there!
+                    warning('Dicom to JSON: Invalid index for t_parms...');
+                    return
+                else
```

Comment:
@jan-petr: Could you help me here? I think there is a bug related to `bMatchDirectories=false`, probably no one really uses it or I am doing something wrong.